service/builder.go

```diff
@@ -22,22 +22,26 @@ package service
 
 import "github.com/pkg/errors"
 
-// WithModules is a helper to create a service without any options
-func WithModules(modules ...ModuleCreateFunc) *Builder {
```

Comment:
Why? I would prefer keeping WithModules.
1.
service.WithModule(
  rpc,
  uhttp,
  task
)

2.
service.WithModule(uhttp)
  .WithModule(rpc)
  .WithModule(task)

1 (the existing way) seems simpler and nicer than 2.